### ISSUE TITLE
`ERA001` does not detect commented out `case` statements

### ISSUE THREAD

**Issue by dosisod** [astral-sh/ruff#10031]
Commented out `case` statements are ignored if they are missing a body, or have a body and fit on one line:

```python
match 1:
    # case 1:          # commented code ignored
    # case 1: print()  # commented code ignored

    case 1:
        pass
        # print()  # commented code found

# if 1: print()  # commented code found

if 1:
    pass
    # print()  # commented code found
```

Running:

```
$ ruff --version
ruff 0.2.2

$ ruff file.py --output-format=concise
file.py:7:9: ERA001 Found commented-out code
file.py:9:1: ERA001 Found commented-out code
file.py:13:5: ERA001 Found commented-out code
```

The first 2 `case` blocks should be counted but aren't. Often times I will duplicate and comment out a `case` statement to figure out why a pattern isn't matching, only to forget to remove the commented out case.

The example with the case/body on one line is less of an issue but still odd, perhaps the `case` statement is preventing the body from being detected.

**Comment by MichaReiser** [comment]
Thanks for the report. 

I think we need to extend the `POSITIVE_CASES` to handle `case` 

Playground: https://play.ruff.rs/5a305aa9-6e5c-4fa4-999a-8fc427ab9a23

**Comment by omh1280** [comment]
@MichaReiser opened my first PR for this! Let me know how it looks
